src/main/java/plcybertrainees/wyjasnienia/Kratnosc.java

```diff
@@ -0,0 +1,14 @@
+package plcybertrainees.wyjasnienia;
+
+public class Kratnosc {
+
+    public Integer dzielenieNaTrzyiPiec(int kratne) {
+
+        if (kratne % 3 % 5 == 0) {
```

Comment:
ten warunek nie zadziała poprawnie np dla liczby 21 która nie jest podzielna przez 3 i 5 
Prawidłowy warunek to:
if (kratne % 3 == 0 && kratne % 5 == 0) {